ibm_catalog.json

```diff
@@ -128,7 +169,129 @@
                   "description": "This architecture supports creating and configuring DB on Cloud resources"
                 }
               ]
-            }
+            },
+            "dependencies": [
+              {
+                "name": "testing-deploy-arch-ibm-observability",
+                "id": "daf2cf91-2384-4205-b7f4-621299a4896d-global",
+                "flavors": [
+                  "standard"
+                ],
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "optional": true,
+                "version": "^1.2.2",
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "resource_group_name",
+                    "version_input": "existing_resource_group_name"
+                  },
+                  {
+                    "value": true,
+                    "dependency_input": "use_existing_resource_group"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "region",
+                    "version_input": "region"
+                  },
+                  {
+                    "reference_version": true,
+                    "dependency_input": "prefix",
+                    "version_input": "prefix"
+                  }
+                ],
+                "ignore_auto_referencing": [
+                  "*"
+                ]
+              },
+              {
+                "name": "deploy-arch-ibm-account-infra-base",
+                "id": "63641cec-6093-4b4f-b7b0-98d2f4185cd6-global",
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "optional": true,
+                "on_by_default": true,
+                "default_flavor": "standard",
+                "version": "^v1.18.2",
+                "input_mapping": [
+                  {
+                    "reference_version": true,
+                    "dependency_input": "prefix",
+                    "version_input": "prefix"
+                  },
+                  {
+                    "dependency_input": "skip_iam_account_settings",
```

Comment:
@ocofaigh do we want to do this always? I think remove this or was it a requirement?

Reply:
matches with https://github.com/terraform-ibm-modules/terraform-ibm-enterprise-app-java/pull/53